src/pages/Animetoast/main.ts

```diff
@@ -0,0 +1,82 @@
+import { pageInterface } from "../pageInterface";
+import { SyncPage } from "../syncPage";
+
+export const Animetoast: pageInterface = {
+    name: "Animetoast",
+    domain: "https://www.animetoast.cc",
+    languages: ['German'],
+    type: "anime",
+    isSyncPage: function (url: string): boolean {
+        return $('.entry-title.light-title').text() != null && url.includes('?link=');
+    },
+    isOverviewPage: function (url: string): boolean {
+        return $('.entry-title.light-title').text() != null && !url.includes('?link=');
+    },
+    sync: {
+        getTitle: function (url: string): string {
+            let title = $('.entry-title.light-title').text().split(' ');
+            if(!(title.length >= 2))return title.join(' ');
+            for(let i = title.length - 1; i >= title.length - 2; i--){
```

Comment:
Better use a regex that removes the parts you dont want with replace then you dont need all this split and loop logic